### PR TITLE
Add error output with custom branch

### PR DESCRIPTION
**Pull request by gaelreyrol**
As some users might encounter a specific error when pushing from a custom branch, I thought it would be useful to quote it so they can search for it.

### CODE REVIEW

File: find-help/troubleshooting.md [hunk 1]
```diff
@@ -30,23 +30,32 @@ For most of Cloud providers, the use of the file system is not a good practice.
...
 
-It may also be because you are working on another branch than master and pushed this specific branch to Clever Cloud.
+It may also be because you are working on another branch than master and pushed this specific branch to Clever Cloud and you encountered this error:
+
+```bash
+remote: You tried to push to a custom branch.
+remote: This is not allowed.
+remote: error: hook declined to update refs/heads/<yourSpecificBranchName>
+...
+error: failed to push some refs to 'xxxx'
```
Comment (BlackYoup) [inline find-help/troubleshooting.md:51]: Why not using `yourSpecificBranchName` here instead of `xxxx`?

**Review by BlackYoup — COMMENTED** [review]
Thanks for the PR, this is a good idea :) I left a comment with a question

**Comment by gaelreyrol** [comment]
Fixed

**Comment by BlackYoup** [comment]
Thanks!

**Pull request merged**